htdocs/public/payment/newpayment.php

```diff
@@ -1536,6 +1573,29 @@
 	print '<input type="hidden" name="tag" value="'.$tag.'">';
 	print '<input type="hidden" name="fulltag" value="'.$fulltag.'">';
 	print '</td></tr>'."\n";
+    
+    if (! empty($conf->stripe->enabled) && empty($object->paid) && ($paymentmethod == 'stripe') ) {
+    require_once DOL_DOCUMENT_ROOT.'/stripe/class/stripe.class.php';	
+    
+    $service = 'StripeLive';
+	$servicestatus = 1;
+  
+	if (empty($conf->global->STRIPE_LIVE) || GETPOST('forcesandbox', 'alpha'))
+	{
+	$service = 'StripeTest';
+	$servicestatus = 0;
+	}
+
+    $stripe = new Stripe($db);
+    $stripeacc = $stripe->getStripeAccount($service);								// Stripe OAuth connect account of dolibarr user (no network access here)
+    $stripecu = $stripe->getStripeCustomerAccount($object->fk_soc, $servicestatus);		// Get thirdparty cu_...
```

Comment:
What happen if the $stripecu is empty (the order is on a customer that has not yet no stripe account) ?

Reply:
i will change it for this function customerStripe() for autocreate stripe customer